src/shared/hooks/usePlugins.ts

```diff
@@ -52,13 +56,14 @@ export default function usePlugins() {
             }, manifest),
           })
         )
-        .catch(error =>
+        .catch(error => {
           setManifest({
             error,
             loading: false,
             data: null,
-          })
-        );
+          });
+          return () => abort.abort();
```

Comment:
AbortController requires you pass in a signal for it to work: [https://developer.mozilla.org/en-US/docs/Web/API/AbortController/abort](https://developer.mozilla.org/en-US/docs/Web/API/AbortController/abort)